clio-client/src/main/scala/org/broadinstitute/clio/client/util/FutureWithErrorMessage.scala

```diff
@@ -7,14 +7,15 @@ import scala.util.{Failure, Success}
 
 trait FutureWithErrorMessage {
   implicit class FutureWithErrorMessage[A](future: Future[A]) {
-    def withErrorMsg(message: String)(implicit ec: ExecutionContext,
-                                      logger: Logger): Future[A] = {
+    def withErrorMsg(message: String = "")(implicit ec: ExecutionContext,
```

Comment:
withErrorMsg should always have a readable message otherwise we should rename it.